bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -193,7 +218,9 @@ class App extends Component {
             {this.renderActionsBar()}
           </div>
           {this.renderUserList()}
+          {userlistIsOpen ? <div className={styles.userlistPad} /> : null}
           {this.renderChat()}
+          {chatIsOpen ? <div className={styles.chatPad} /> : null}
```

Comment:
I did not understand the use of these pads.

Reply:
<div className={styles.chatPad} /> can be removed,  but the <div className={styles.userlistPad} /> needs to be there so the re-size handle doesn't disappear when both the userlist and chat are open

Reply:
I'll see if i can remove the `<div className={styles.userlistPad} />` as well and use the z-index to fix the issue.